cypress.json

```diff
@@ -1 +1,4 @@
-{}
+{
+  "baseUrl": "http://localhost:8080",
+  "defaultCommandTimeout": 7000
```

Comment:
Sets default command timeout like for cy.get/contains, etc to 7s, by default cy.visit is like 60s, we add the baseUrl so you don't always have to do http://localhost:8080 prefix for every path